.vscode/settings.json

```diff
@@ -1,75 +1,92 @@
 {
-	"[rust]": {
-		"editor.formatOnSave": true
-	},
-	"[starlark]": {
-		"editor.formatOnSave": true
-	},
-	"emmet.includeLanguages": {
-		"rust": "html"
-	},
-	"[javascript][javascriptreact][typescript][typescriptreact][github-actions-workflow]": {
-		"editor.defaultFormatter": "biomejs.biome",
-		"editor.codeActionsOnSave": {
-			"quickfix.biome": "explicit",
-			"source.organizeImports.biome": "explicit"
-		},
-		"editor.formatOnSave": true
-	},
-	"biome.rename": true,
-	"biome.enabled": true,
-	"rust-analyzer.cargo.extraEnv": {
-		"RISC0_SKIP_BUILD": "1"
-	},
-	"files.eol": "\n",
-	"git.ignoreLimitWarning": true,
-	"files.associations": {
-		"*.sh": "shellscript",
-		"*.py": "python",
-		"*.cl": "c",
-		"*.rs": "rust",
-		"*.BUILD": "starlark",
-		"meson.build": "meson",
-		"*.mlir": "mlir",
-		"*.pbtxt": "prototxt",
-		"*.cfg": "python",
-		"*.scr": "tcl",
-		"cstddef": "cpp",
-		"*.bazel": "starlark",
-		"fstream": "cpp",
-		"ios": "cpp",
-		"iosfwd": "cpp",
-		"vector": "cpp",
-		"charconv": "cpp",
-		"stdexcept": "cpp",
-		"array": "cpp",
+  "[rust]": {
```

Comment:
@Cardosaum were these changes intentional?

Reply:
Oh.. My vscode probably auto-formatted the file, sorry for not catching that.

The only changes I spotted there are two new entries to `cSpell.words`. Would it be okay to leave as is (would we have a default formatter for json?)